RockWeb/Blocks/Finance/TransactionEntry.ascx.cs

```diff
@@ -312,6 +324,55 @@ protected override void OnLoad( EventArgs e )
             pnlDupWarning.Visible = false;
             nbSaveAccount.Visible = false;
 
+            string accountParameterType = string.Empty;
+            using ( var rockContext = new RockContext() )
+            {
+                if ( !string.IsNullOrWhiteSpace( PageParameter( "account" ) ) )
```

Comment:
@treyhendon it might be better to have the parameter name be 'AccountIds' instead of 'account'